### PR TITLE
Adaptive App Icon

### PR DESCRIPTION
**Pull request by zutrinken**
* Add Adaptive Icon
* Update Launch Screen Logo
* Update Logo in Navigation
* Update Notification Icon

### CODE REVIEW

**Comment by stephanlindauer** [comment]
* i changed the PR base to branch develop
* build still fails with
```
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Android resource linking failed
  /home/travis/build/criticalmaps/criticalmaps-android/app/src/main/res/drawable/ic_logo.xml:15: AAPT: error: '#@color/colorSecondary' is incompatible with attribute fillColor (attr) color.
```

**Comment by cbalster** [comment]
@zutrinken are you okay with me pushing a commit to this branch fixing the issues mentioned? then we can get this merged an I can add another PR.

**Comment by cbalster** [comment]
Also we can also get rid of the different dpi versions of `ic_launcher_foreground.png` by using a single VectorDrawable. This shouldn't be problem since it doesn't have any gradients. If you'd like I can take care of that if you can supply me an .svg or .ai.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <application
         android:allowBackup="true"
         android:name=".App"
-        android:icon="@mipmap/ic_launcher_48dp"
+        android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
```
Comment (cbalster) [inline app/src/main/AndroidManifest.xml]: `android:roundIcon="@mipmap/ic_launcher_round"` is missing here for devices < API 26 and launchers supporting it

**Comment by zutrinken** [comment]
@cbalster please take over! I'll slack you the .svg

**Comment by cbalster** [comment]
should be good to go

**Pull request merged**